src/views/pools/utils.ts

```diff
@@ -0,0 +1,56 @@
+import { baseAmount, formatBaseAsTokenAmount } from '@thorchain/asgardex-token'
+import { validBNOrZero, bn } from '@thorchain/asgardex-util'
+
+import { getAssetFromString } from '../../helpers/tokenHelpers'
+import { PriceDataIndex } from '../../services/midgard/types'
+import { PoolDetail } from '../../types/generated/midgard'
+import { PoolDataType } from './types'
+
+export const getPoolData = (
```

Comment:
Please add a test for this function `getPoolData`, we already have a test in BEPSwap for it (see https://gitlab.com/thorchain/bepswap/bepswap-web-ui/blob/b514aad8ffdc9d4ffd41c8b5c380cfc09d2f1df0/src/containers/Pool/utils.test.ts#L184)

Reply:
Issue https://github.com/thorchain/asgardex-electron/issues/158